common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerValidator.java

```diff
@@ -229,6 +231,26 @@ private X509Certificate getSelfSignedCert(final List<X509Certificate> certs)
         return selfSignedCert;
     }
 
+    /**
+     * Returns the package that is currently active relative to the Work Account custom account type
+     * Note: either the company portal or the authenticator
+     *
+     * @param context : current active broker package name, null if no broker is available
```

Comment:
Nit: Should this be the `@return` ? Seems like this content on the wrong attribute

Reply:
Addressed via https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/918/commits/477877f2d99d6eb7d752f18d941a682b51665c62